feature-libs/storefinder/core/facade/store-data.service.ts

```diff
@@ -79,17 +79,29 @@ export class StoreDataService {
    */
   protected getSchedule(location: PointOfService, date: Date): any {
     const weekday = this.weekDays[date.getDay()];
+    const language = this.winRef.sessionStorage?.getItem('language');
     return location.openingHours.weekDayOpeningList.find(
       (weekDayOpeningListItem) =>
         weekDayOpeningListItem.weekDay ===
-          date.toLocaleString(window.sessionStorage.language, {
+          date.toLocaleString(language as any, {
```

Comment:
What happens if you pass `null` to the `toLocaleString` function? Seems like it takes a string or array of strings.

Do a null check before so that use know language is a `string`.